### PR TITLE
show badges for master branch

### PR DESCRIPTION
**Pull request by bernt-matthias**
and use simpler URLs

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [x] - This PR does something else (explain below)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-[![Galaxy Tool Linting and Tests for push and PR](https://github.com/galaxyproject/tools-iuc/workflows/Galaxy%20Tool%20Linting%20and%20Tests%20for%20push%20and%20PR/badge.svg)](https://github.com/galaxyproject/tools-iuc/actions?query=workflow%3A%22Galaxy+Tool+Linting+and+Tests+for+push+and+PR%22)
-[![Weekly global Tool Linting and Tests](https://github.com/galaxyproject/tools-iuc/workflows/Weekly%20global%20Tool%20Linting%20and%20Tests/badge.svg)](https://github.com/galaxyproject/tools-iuc/actions?query=workflow%3A%22Weekly+global+Tool+Linting+and+Tests%22)
+[![Galaxy Tool Linting and Tests for push and PR](https://github.com/galaxyproject/tools-iuc/actions/workflows/pr.yaml/badge.svg?branch=master)](https://github.com/galaxyproject/tools-iuc/actions/workflows/pr.yaml/badge.svg)
```
Comment (mvdbeek) [inline README.md:1]:
```suggestion
[![Galaxy Tool Linting and Tests for push and PR](https://github.com/galaxyproject/tools-iuc/actions/workflows/pr.yaml/badge.svg?branch=master&event=push)](https://github.com/galaxyproject/tools-iuc/actions/workflows/pr.yaml/badge.svg)
```

**Review by mvdbeek — APPROVED** [review]

**Comment by bernt-matthias** [comment]
@mvdbeek Do you know why the combine step does not finish? Or better: is not shown as finished.

**Comment by mvdbeek** [comment]
No idea ... also I don't know why I can't just merge it, I should have admin rights on this repo.

**Comment by mvdbeek** [comment]
I'm just gonna push it to master, probably some weird bug on the github interface

**Pull request merged**